lua/nvim-tree/explorer/node-builders.lua

```diff
@@ -42,13 +49,20 @@ end
 
 function M.file(absolute_path, name, status, parent_ignored)
   local ext = string.match(name, ".?[^.]+%.(.*)") or ""
+  local stat = uv.fs_stat(absolute_path)
+
+  local last_modified = 0
+  if stat ~= nil then
+    last_modified = stat.mtime.sec
+  end
```

Comment:
duplicated code, could you extract this in a get_last_modified function or something like that ?